cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/KafkaMetricAnomaly.java

```diff
@@ -65,7 +65,7 @@ public boolean fix() throws KafkaCruiseControlException {
 
   @Override
   public String toString() {
-    return String.format("%s anomaly with id: %s. Anomaly description: %s", METRIC_ANOMALY, anomalyId(), _description);
+    return String.format("[%s] %s", anomalyId(), _description);
```

Comment:
Nit: Can we consider formatting it as `format("[anomaly ID=%s] %s", anomalyId(), _description);`? I think this way is slightly more descriptive.

Reply:
Anomaly ID is a UUID (i.e. `d1c17515-8cb6-4ecc-8079-e57425273f6b`) -- from the format, it should be obvious that this is an anomaly id. I prefer conciseness to to help with readability in a single page.

Reply:
My concern is that when users look at this ID in the log file, they have no idea what ID is it since there are other types of IDs that are also in this UUID format right? I guess users can figure out that it is an anomaly ID by reading the description. IMO, add several characters `anomaly ID=` does not harm the conciseness that much since it is much shorter than the UUID itself.

Feel free to drop this comment if you still think the change is not necessary.